### ISSUE TITLE
Create a single model for the site-wide surveys.

### ISSUE THREAD

**Issue by csmcallister** [18F/10x-MLaaS#38]
## 10x Qualitative Data User story 
As the project's data scientist, I want a single model instead of 4 for the site-wide survey so that the app performs efficiently without taking up a lot of disk space. Ideally, we won't have to use word embeddings, which take up a lot of disk space and increase compute time for training.
  
## Acceptance criteria
- [x] Hand-label new training data (with all comments concatenated)
- [ ] Have someone peer review the hand-labeled training data
- [x] Create a ham-spam classifier that performs better than [dummy classifiers](http://scikit-learn.org/stable/modules/generated/sklearn.dummy.DummyClassifier.html) 
- [x] Put the model in`eda/` for review
